db/migrations/20231219030949_add_msat_total_to_albums_podcasts.js

```diff
@@ -0,0 +1,23 @@
+exports.up = function (knex) {
+  return knex.schema
+    .alterTable("podcast", function (table) {
+      table.bigInteger("msat_total").notNullable().defaultTo(0);
+      table.check("msat_total >= 0", [], "msat_total_check");
```

Comment:
It'll help debug if these constraint names are unique
```suggestion
      table.check("msat_total >= 0", [], "podcast_msat_total_check");
```